outputs/webhook.go

```diff
@@ -29,7 +30,8 @@ func (webhook *WebhookOutput) Init() error {
 func (webhook *WebhookOutput) Send(content map[string]string) error {
 	log.Printf("Sending webhook to %q", webhook.Url)
 	data := content["description"] //it's not supposed to work with legacy renderer
-	resp, err := http.Post(webhook.Url, "application/json", strings.NewReader(data))
+	client := http.Client{Timeout: time.Duration(120) * time.Second}
```

Comment:
I would make it configurable with default of 2 minutes

Reply:
One idea you can implement is to dependency inject this in by making it a configurable option like @AndreyLevchenko mentioned on the WebhookOutput struct. That way in a test  you can configure it and test the case where timeout actually happens.